packages/app/src/pages/Specs/Runner.vue

```diff
@@ -72,3 +75,50 @@ window.__showSelectorPlaygroundForTestingPurposes = () => {
     }
   }
 </route>
+
+<style lang="scss">
+
+iframe.aut-iframe {
+  width: 100%;
+  height: 100%;
+  background: white;
+}
+
+iframe.spec-iframe {
+    border: none;
+    height: 0;
+    position: absolute;
+    visibility: hidden;
+    width: 0;
+}
+
+.highlight {
+  background: rgba(159, 196, 231, 0.6);
+  border: solid 2px #9FC4E7;
+  cursor: pointer;
+}
+
+.tooltip {
+  font-family: sans-serif;
+  font-size: 14px;
+  max-width: 400px !important;
+}
+
+#unified-runner {
+  position: relative;
+    margin: 0 auto;
+}
+
+.is-screenshotting #unified-runner {
+    margin: unset;
+}
+
+#unified-runner > .screenshot-height-container {
+  height: 100%
```

Comment:
Should this be dynamic based on the user's defined screen height?

Reply:
We can do this as a separate PR since this one is mainly to unblock develop, just wanted to note it

Reply:
Do you mean viewport height? If so there's a `viewportStyle` implemented separately that handles that. This rule is only targeting the wrapper div that sometimes, for screenshots, needs to handle height differently.

Reply:
Ah gotcha